src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -41,13 +41,6 @@ export class HostsComponent implements OnInit, OnDestroy {
         pipe: this.cephShortVersionPipe
       }
     ];
-    this.interval = setInterval(() => {
```

Comment:
Please remove the `interval` field from this component (line 16)

Reply:
done